textworld/challenges/spaceship/spaceship_game.py

```diff
@@ -0,0 +1,411 @@
+import argparse
+import os
+
+from os.path import join as pjoin
+from typing import Mapping, Optional
+
+import textworld
+from textworld import g_rng
+from textworld import GameMaker
+from textworld.challenges import register
+from textworld.generator.data import KnowledgeBase
+from textworld.generator.game import GameOptions, Event, Quest
+
+
+g_rng.set_seed(20190826)
+PATH = pjoin(os.path.dirname(__file__), 'textworld_data')
+
+
+def build_argparser(parser=None):
+    parser = parser or argparse.ArgumentParser()
+
+    group = parser.add_argument_group('Spaceship game settings')
+    group.add_argument("--level", required=True, choices=["easy", "balanced", "sparse"],
```

Comment:
According to your code, the choices should be
balanced -> medium
sparse -> difficult

Reply:
This has already been fixed on the code.